experimental/PropertyDDS/packages/property-changeset/src/changeset.js

```diff
@@ -6,25 +6,21 @@
  * @fileoverview Serialized representation of the changes in a repository
  */
 
-const _ = require('lodash');
+import { cloneDeep as deepCopy, isString, isObject, isArray, isEmpty, extend, keys, each } from "lodash";
```

Comment:
Importing lodash as '_' seems to prevent tree shaking (see issue #6971).

According to [this article](https://medium.com/@martin_hotell/tree-shake-lodash-with-webpack-jest-and-typescript-2734fa13b5cd) `import { ... } from "lodash"` appears to be the best option.

(Issue #6971 suggests importing via internal modules (e.g., 'lodash/isNumber'), but reaching into a modules internal file structure is non-ideal.)

Reply:
(I'll go ahead and merge as-is and we can follow up on bundle size separately.)

Reply:
yes, I discussed these advantages briefly with @ruiter and we decided to keep it consistent with other packages for now.